### PR TITLE
[ACS-4118] create a tag from tags list

### PR DESCRIPTION
**Pull request by AleksanderSklorz**
**Please check if the PR fulfills these requirements**
```
[x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-js-api/wiki/Commit-format)
[ ] Tests for the changes have been added (for bug fixes / features)
[x] Docs have been added / updated (for bug fixes / features)
```

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/ACS-4118

**What is the new behavior?**
Added function to create tags without node and to load specific tag by name. 

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
ADF PR https://github.com/Alfresco/alfresco-ng2-components/pull/8143
APPS PR https://github.com/Alfresco/alfresco-apps/pull/4759

### CODE REVIEW

File: src/api/content-rest-api/docs/TagsApi.md [hunk 1]
```diff
@@ -447,3 +449,38 @@ parameter are returned in addition to those specified in the **fields** paramete
...
+const alfrescoApi = new AlfrescoApi({
+    hostEcm: 'http://127.0.0.1:8080'
+});
+
+const tagsApi = new TagsApi(alfrescoApi);
+
+tagsApi.createTags(tags).then((data) => {
+  console.log('API called successfully. Returned data: ' + data);
+}, function(error) {
+  console.error(error);
```
Comment (jatin2008) [inline src/api/content-rest-api/docs/TagsApi.md:472]: can we console the error here, well am not sure.

File: src/api/content-rest-api/docs/TagsApi.md [hunk 2]
```diff
@@ -447,3 +449,38 @@ parameter are returned in addition to those specified in the **fields** paramete
...
+import { AlfrescoApi, TagsApi} from '@alfresco/js-api';
+
+const alfrescoApi = new AlfrescoApi({
+    hostEcm: 'http://127.0.0.1:8080'
+});
+
+const tagsApi = new TagsApi(alfrescoApi);
+
+tagsApi.createTags(tags).then((data) => {
+  console.log('API called successfully. Returned data: ' + data);
```
[the remark this answers is not in the document]
Comment (AleksanderSklorz) [reply src/api/content-rest-api/docs/TagsApi.md:470]: It is only documentation file. That code is not executed. It's only example. We have similar examples for other places for example: 
image

File: src/api/content-rest-api/docs/TagsApi.md (hunk 1 above)
Comment (AleksanderSklorz) [reply src/api/content-rest-api/docs/TagsApi.md:472]: It is only documentation file. That code is not executed. It's only example. We have similar examples for other places for example:
image

**Review by MichalKinas — APPROVED** [review]

**Review by DenysVuika — APPROVED** [review]

File: src/api/content-rest-api/docs/TagsApi.md (hunk 1 above)
Comment (jatin2008) [reply src/api/content-rest-api/docs/TagsApi.md:472]: then it's fine, no issues in it.

File: src/api/content-rest-api/docs/TagsApi.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jatin2008) [reply src/api/content-rest-api/docs/TagsApi.md:470]: its fine then.

**Review by jatin2008 — APPROVED** [review]

**Pull request merged**